src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -389,59 +378,27 @@ protected final void verifyWithLimitedResources(Configuration aConfig,
             throws Exception {
         // We return null here, which gives us a result to make an assertion about
         final Void result = TestUtil.getResultWithLimitedResources(() -> {
-            verifyWithInlineConfigParser(aConfig, fileName, expected);
+            verifyWithInlineConfigParser(fileName, expected);
```

Comment:
@shashwatj07 @romani How are we verifying a configuration run without providing it said configuration?

Reply:
I feel more comfortable, for now, its not breaking tests by the new issue I created.

This can be a simple parameter removal in a new PR.

Reply:
I get your point. I wonder why CI didn't complain of unused parameter. I'll remove it.
I will send PR to throw proper exception.

Reply:
I am not sure about PMD or TeamCity, but Checkstyle itself has no check to catch something like this. I only found it because my Eclipse configuration is set to produce more errors on things like. We can't enable it in CI because it produces false exceptions that can't be suppressed.

Reply:
I'm suspecting PMD. Although, I get your point. Thank you.

Reply:
https://github.com/checkstyle/checkstyle/pull/10767

Reply:
@pbludov You know PMD best. PMD has `UnusedFormalParameter`. Do we have it enabled and should it have caught this?
https://www.ing.iac.es//~docs/external/java/pmd/rules/unusedcode.html